### ISSUE TITLE
E-Mail verification is not send

### ISSUE THREAD

**Issue by FWuellhorst** [pypa/warehouse#9857]
**Describe the bug**
I don't receive any verification e-mail for my account (`erc_ebc`). 
I tried two different e-mail addresses and hit the resend verification button a few times.

**Expected behavior**
I should receive the e-mail to upload a project.

**My Platform**
I use Firefox, but this should not matter. 
The mail addresses use gmx and ms-exchange as server.

**Additional context**
Maybe related: https://github.com/pypa/pypi-support/issues/566

**Comment by ewjoachim** [comment]
I'm seeing you reverify requests in your profile:
![Screenshot from 2021-08-02 12-39-48](https://user-images.githubusercontent.com/1457576/127849336-f84ff9ac-65f6-4804-9613-fca6e92441a8.png)
But there's no "sent email" event

In the PyPI Moderator email section, at the time of your reverify requests, I don't see an email pertaining to that, so if the email
![Screenshot from 2021-08-02 12-40-00](https://user-images.githubusercontent.com/1457576/127849578-c305ec1f-512a-47c9-af99-2e3ca613ea35.png)

So, long story short : no idea, we may need an admin to look at it.

**Comment by FWuellhorst** [comment]
Thanks for the quick reply! Do I have to contact an admin or will one automatically look at this?

**Comment by ewjoachim** [comment]
They'll look at this :)

**Comment by di** [comment]
The underlying issue seems to be the comma in this user's display name. Removing it should allow the emails to send.

I briefly attempted to debug this in https://github.com/pypa/warehouse/compare/fix/9857 by adding some tests but our behavior seems to be correct. We also have no exceptions or errors, either from warehouse or from SES, regarding these emails.

**Comment by di** [comment]
The underlying issue actually appears to be https://bugs.python.org/issue44637

**Comment by FWuellhorst** [comment]
Nice, thank you very much for the quick support! Removing the comma works. 
Should I keep the issue open as new users don't receive a warning about invalid display names?

**Comment by ewjoachim** [comment]
Yes, your case is solved but the underlying problem is still there.

**Comment by ewjoachim** [comment]
On the Python issue, someone said:
> it works fine with the `compat32` policy

I believe this could make a one-liner fix for this issue while waiting for a more definitive fix.
